java/src/client/facade/FacadeManager.java

```diff
@@ -0,0 +1,65 @@
+package client.facade;
+
+import shared.models.game.ClientModel;
+
+/**
+ * Created by audakel on 9/17/16.
```

Comment:
it actually depends. its really a style choice, and with many members of the team not super familiar with git, it makes sense to add a author tag for easy reference in case the have questions about the file.

http://programmers.stackexchange.com/a/126263